src/components/visualizations/chart/highcharts/helpers.ts

```diff
@@ -137,3 +155,71 @@ export function getDataLabelAttributes(point: any) {
         height: 0
     };
 }
+
+export function shouldFollowPointer(chartOptions: any) {
+    // Sets true if max or min is set on any axis from properties
+    // and is lower than max data value from series
+    const type = chartOptions.type;
+
+    if (!isColumnChart(type) && !isBarChart(type)) {
+        return false;
+    }
+
+    const xMax = Number(get(chartOptions, 'xAxisProps.max'));
+    const yMax = Number(get(chartOptions, 'yAxisProps.max'));
+    const xMin = Number(get(chartOptions, 'xAxisProps.min'));
+    const yMin = Number(get(chartOptions, 'yAxisProps.min'));
+
+    // Return false when no extrem is set in properties
+    if (!xMax && !yMax && !xMin && !yMin) {
+        return false;
+    }
+
+    if (isBarChart(type) && !xMax && !xMin) {
+        return false;
+    }
+
+    if (!isBarChart(type) && !yMax && !yMin) {
```

Comment:
`!isColumnChart`?

Reply:
Yes, I want this follow Pointer only for Column chart and Bar chart, because on other (Line chart) is tooltip positioned on data point which is visible within axis range.